content/learn/manipulating-files/node-file-descriptors.pt.md

```diff
@@ -0,0 +1,102 @@
+---
+title: working-with-file-descriptors-in-nodejs
+displayTitle: 'Trabalhando com os descritores de ficheiro na Node.js'
+description: 'Como interagir com os descritores de ficheiro usando a Node.js'
+authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais, clean99, vaishnav-mk, nazarepiedady
+category: learn
+---
+
+Antes de seres capaz de interagir com um ficheiro que encontra-se no teu sistema de ficheiro, deves ter um descritor de ficheiro.
+
+Um descritor de ficheiro é uma referência à um ficheiro aberto, um número (`fd`, descritor de ficheiro) retornado pela abertura do ficheiro usando o método `open()` oferecido pelo módulo `fs`. Este número (`fd`, ou descritor de ficheiro) unicamente identifica um ficheiro aberto no sistema operacional:
+
+```js
+const fs = require('fs');
+
+fs.open('/Users/joe/test.txt', 'r', (err, fd) => {
+  // `fd` é o nosso descritor de ficheiro
+});
+```
+
+Repara a opção `r` que usamos como segundo parâmetro para a chamada de `fs.open()`.
+
+Aquela opção significa que abrimos o ficheiro para leitura.
+
+**Outras opções que comummente usarás são:**
+
+<table>
+  <tr>
+    <th>Opção</th>
+    <th>Descrição</th>
+    <th>O ficheiro é criado se não existir</th>
+  </tr>
+  <tr>
+    <td><code>r+</code></td>
+    <td>Esta opção abre o ficheiro para <b>leitura</b> e <b>escrita</b></td>
+    <td style="text-align: center;">❌</td>
+  </tr>
+  <tr>
+    <td><code>w+</code></td>
+    <td>Esta opção abre o ficheiro para <b>leitura</b> e <b>escrita</b> e também posiciona o lina no <b>princípio</b> do ficheiro</td>
```

Comment:
```suggestion
    <td>Esta opção abre o ficheiro para <b>leitura</b> e <b>escrita</b> e também posiciona a linha no <b>início</b> do ficheiro</td>
```

Reply:
The change to be made is "posiciona o" to "posiciona-o" the rest should remain as it is. I will do it now.